cmd/kazel/kazel.go

```diff
@@ -351,6 +351,16 @@ func writeFile(path string, f *build.File, boilerplate []byte, exists, dryRun bo
 	build.Rewrite(f, &info)
 	var out []byte
 	out = append(out, boilerplate...)
+	// double format the source file as our modification logic sometimes uses
+	// LiteralExpr where it should use other types of expressions, and this
+	// prevents issues where kazel thus formats structures incorrectly.
+	// :this_is_fine:
+	outData := build.Format(f)
```

Comment:
Hilarious hack... We should do a fix sometime soon, but happy to unblock this work.